apps/src/code-studio/pd/application_dashboard/summary.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Application Dashboard summary view.
+ * Route: /summary
+ */
+import React from 'react';
+import SummaryTable from './summary_table';
+
+export default class Summary extends React.Component {
+  render() {
+    return (
+      <div>
+        <h1>All Regional Partner Applications</h1>
+        <SummaryTable caption="CSD Facilitators"/>
+        <SummaryTable caption="CSP Facilitators"/>
+        <SummaryTable caption="CSD Teachers"/>
+        <SummaryTable caption="CSP Teachers"/>
```

Comment:
Again, like above, this is fine for an initial skeleton / demo, but we'll need to pass more props here so these summary tables know what data to display.

Reply:
Yes, this is just to get the skeleton out there. additional props will be coming in a followup